test/Error/Either.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE ScopedTypeVariables, TypeApplications #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Error.Either
+( tests
+) where
+
+import Control.Carrier.Error.Either
+import Control.Monad.Trans.Except
+import Pure
+import Test.QuickCheck.Poly
```

Comment:
This gives us convenient `A`, `B`, and `C` types with `Arbitrary` instances already attached.